### PR TITLE
Dev wip

### PR DESCRIPTION
**Pull request by dmipx**
Merged d4xx IPU6 version.
Renaming motion_t to ir
Camera devicetree as source, not patch.
Static analisys fixes.
Dedicated sensor controls.

### CODE REVIEW

**Comment by Nir-Az** [comment]
Please make sure the Jenkins scripts fit this changes and can build this branch, if not adjustment PR will be needed together with this one.
i.e.
https://github.com/IntelRealSense/librealsense-deploy/blob/0d64153900571cf6285c699ad0d5cb91a2f8c607/jenkins/LRS_Jetson_mipi_usb_driver_jp4_jp5.groovy#L204

File: hardware/realsense/tegra194-camera-d4xx-generic.dtsi
[the remark this answers is not in the document]
Comment (dmipx) [reply hardware/realsense/tegra194-camera-d4xx-generic.dtsi:1]: Thanks fo your review.
I will rework dtsi for single camera, with "generic" there is i2c-mux regs that differs than "single". Our adapter have not all i2c-mux ports shorted like dual GMSL card.

File: kernel/realsense/d4xx.c [hunk 1]
```diff
@@ -2034,6 +2564,23 @@ static int ds5_g_volatile_ctrl(struct v4l2_ctrl *ctrl)
...
+	case V4L2_CID_IPU_QUERY_SUB_STREAM: {
+		if (sensor) {
+			int vc_id = get_sub_stream_vc_id(pad_to_substream[sensor->mux_pad]);
+
+			dev_dbg(sensor->sd.dev,
+				"%s(): V4L2_CID_IPU_QUERY_SUB_STREAM sensor->mux_pad:%d vc:[%d]\n",
+				__func__, sensor->mux_pad, vc_id);
+			*ctrl->p_new.p_s32 = pad_to_substream[sensor->mux_pad];
+			state->mux.last_set = sensor;
+			} else {
```
[the remark this answers is not in the document]
Comment (dmipx) [reply kernel/realsense/d4xx.c]: done.

File: README.md [hunk 2]
```diff
@@ -53,7 +53,10 @@ Apply D457 patches and build the kernel image, dtb and D457 driver.
 # or, if using direct download method
 # ./apply_patches_ext.sh [--one-cam | --dual-cam] ./Linux_for_tegra/source/public [JetPack_version]
 
-Note: The `--one-cam` and `--dual-cam` option applies only for JetPack 5.0.2. By setting the `--one-cam` option it builds DT with only camera on GMSL link A. Without it the default is to build dual camera configuration for JetPack 5.0.2.
+Note: The `--one-cam` and `--dual-cam` option applies only for JetPack 5.0.2 compatible with adapter: https://store.intelrealsense.com/buy-intel-realsense-des457.html.
+By setting the `--one-cam` option it builds DT with only camera on GMSL link A.
+By setting the `--dual-cam` option it builds DT with dual cameras on GMSL link A.
```
[the remark this answers is not in the document]
Comment (dmipx) [reply README.md]: corrected.

**Pull request merged**